medcat/config_meta_cat.py

```diff
@@ -37,6 +37,7 @@ class General(MixingConfig, BaseModel):
     a deployment."""
     pipe_batch_size_in_chars: int = 20000000
     """How many characters are piped at once into the meta_cat class"""
+    span_group: Optional[str] = None
```

Comment:
Would be great to add a docstring below - like the rest of them - to know what value is expected here.